lib/helpers.bash

```diff
@@ -951,6 +951,24 @@ function pathmunge() {
 	fi
 }
 
+function _remote_name() {
+	local branch
+	branch=$(git branch --show-current)
```

Comment:
Doesn't this get the information about the git repository of the current working directory? Shouldn't this get the information in the Bash-it repository?